packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -657,16 +658,33 @@ export class PeerManager {
    */
   private onLibp2pPeerDisconnect = (evt: CustomEvent<Connection>): void => {
     const {direction, status, remotePeer} = evt.detail;
+    const peerIdStr = remotePeer.toString();
+
+    let logMessage = "A peer disconnected us";
```

Comment:
pretty sure callback is triggered both when they disconnect us and when we disconnect them